### PR TITLE
Spotify

### PR DESCRIPTION
**Pull request by ArgoNavis1763**
Login and logout functionality working as well as all tests passing

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,3 @@
-REACT_APP_BAND_BUDDY_API_URL=http://localhost:4000
\ No newline at end of file
+REACT_APP_BAND_BUDDY_API_URL=http://localhost:3001
```
Comment (TomGroombridge) [inline .env]: please delete this whole file. We don't want .env in github

File: src/Components/RecordingButton/index.jsx [hunk 2]
```diff
@@ -16,11 +16,13 @@ export const RecordingButton = (props) => {
         props.setTitle("");
         props.setArtwork("");
         props.setSongNotFound(false);
-        // props.setSpotifyID("")
+        // props.setSpotifyArtistID("")
```
Comment (TomGroombridge) [inline src/Components/RecordingButton/index.jsx:19]: remove comments

File: src/Pages/Home/index.jsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  const [artist, setArtist] = useState('')
+  const [artwork, setArtwork] = useState()
+  const [songNotFound, setSongNotFound] = useState(false)
+  const [cardState, setCardState] = useState('songCard')
+  const [APIError, setAPIError] = useState(false)
+  const [loading, setLoading] = useState(false)
+  const [cookies, setCookie, removeCookie] = useCookies(["accessToken"])
+  let navigate = useNavigate();
+
+  // const [spotifyArtistID, setSpotifyArtistID] = useState("")
```
Comment (TomGroombridge) [inline src/Pages/Home/index.jsx:26]: remove

File: src/Pages/Home/index.jsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+          setTitle={setTitle}
+          setAlbum={setAlbum}
+          setArtist={setArtist}
+          setArtwork={setArtwork}
+          setSongNotFound={setSongNotFound}
+          setCardState={setCardState}
+          setAPIError={setAPIError}
+          setLoading={setLoading}
+          loading={loading}
+          // setSpotifyArtistID={setSpotifyArtistID}
```
Comment (TomGroombridge) [inline src/Pages/Home/index.jsx:69]: remove

File: src/Pages/Home/index.jsx [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+            album={album}
+            artist={artist}
+            artwork={artwork}
+          />
+        ) : null}
+
+        {title !== '' && cardState === 'artistCard' ? (
+          <ArtistCard
+            artist={artist}
+            // spotifyArtistID={spotifyArtistID}
```
Comment (TomGroombridge) [inline src/Pages/Home/index.jsx:92]: remove

File: src/index.js [hunk 6]
```diff
@@ -1,17 +1,20 @@
...
+import { CookiesProvider } from 'react-cookie'
+import './index.css'
+import App from './App'
+import reportWebVitals from './reportWebVitals'
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+const root = ReactDOM.createRoot(document.getElementById('root'))
 root.render(
-  <React.StrictMode>
+  // <React.StrictMode>
```
Comment (TomGroombridge) [inline src/index.js]: remove

**Review by TomGroombridge — COMMENTED** [review]
Looks great can we just please not commit the .env file

File: src/Components/RecordingButton/index.jsx (hunk 2 above)
Comment (ArgoNavis1763) [reply src/Components/RecordingButton/index.jsx:19]: Comments are prep for the spotify stuff we'll be working on this week

File: src/Pages/Home/index.jsx (hunk 3 above)
Comment (ArgoNavis1763) [reply src/Pages/Home/index.jsx:26]: Comments are prep for the spotify stuff we'll be working on this week

File: src/Pages/Home/index.jsx (hunk 4 above)
Comment (ArgoNavis1763) [reply src/Pages/Home/index.jsx:69]: Comments are prep for the spotify stuff we'll be working on this week

File: src/Pages/Home/index.jsx (hunk 5 above)
Comment (ArgoNavis1763) [reply src/Pages/Home/index.jsx:92]: Comments are prep for the spotify stuff we'll be working on this week

File: src/index.js (hunk 6 above)
Comment (ArgoNavis1763) [reply src/index.js]: removed

File: .env (hunk 1 above)
Comment (ArgoNavis1763) [reply .env]: done

**Pull request merged**